system-tests/management-api/management-api-test-runner/src/test/java/org/eclipse/edc/test/e2e/managementapi/ContractDefinitionApiEndToEndTest.java

```diff
@@ -117,6 +123,44 @@ void queryPolicyDefinitionWithSimplePrivateProperties(ManagementEndToEndTestCont
                     .body("size()", is(0));
         }
 
+        @Test
+        void queryContractDefinitions_sortByCreatedDate(ManagementEndToEndTestContext context, ContractDefinitionStore store) throws JsonProcessingException {
+            var id1 = UUID.randomUUID().toString();
+            var id2 = UUID.randomUUID().toString();
+            var id3 = UUID.randomUUID().toString();
+            Stream.of(id1, id2, id3).forEach(id -> store.save(createContractDefinition(id).createdAt(System.nanoTime()).build()));
```

Comment:
Please don't do this. Start with a time and the increment using an offset so it is deterministic.